stable_diffusion/stable_diffusion.ipynb

```diff
@@ -167,7 +170,6 @@
     "            except OSError:\n",
     "                pipelines[m] = init_model(m, torch_dtype=torch.float16, local_files_only=False)\n",
     "            \n",
-    "        \n",
     "    if torch.cuda.is_available():\n",
     "        pseudo_rand_gen = torch.Generator(device='cuda')\n",
```

Comment:
Use cuda only if available. This should allow the cells to run on a cpu machine.